src/reducers/versions.spec.tsx

```diff
@@ -121,6 +121,15 @@ describe(__filename, () => {
       );
     });
 
+    it('throws an error when updateSelectedPath is called for an unknown version', () => {
+      expect(() => {
+        reducer(
+          undefined,
+          actions.updateSelectedPath({ selectedPath: 'pa/th', versionId: 123 }),
+        );
+      }).toThrow();
```

Comment:
In a lot of our tests we assert on the actual error message. Maybe that's overkill (and introduces a maintenance burden if the messages change), but I wanted to raise it here as an inconsistency. This same comment applies to all the `expect(...).toThrow()` tests.

Reply:
Yeah, since so many errors *could* be thrown, I think it's always helpful to assert some part of the message.